.github/workflows/ci.yml

```diff
@@ -243,25 +250,129 @@ jobs:
           BITCOINUTIL: '${{ github.workspace }}\build\bin\Release\bitcoin-util.exe'
           BITCOINWALLET: '${{ github.workspace }}\build\bin\Release\bitcoin-wallet.exe'
           TEST_RUNNER_EXTRA: ${{ github.event_name != 'pull_request' && '--extended' || '' }}
-        shell: cmd
-        run: py -3 test\functional\test_runner.py --jobs %NUMBER_OF_PROCESSORS% --ci --quiet --tmpdirprefix=%RUNNER_TEMP% --combinedlogslen=99999999 --timeout-factor=%TEST_RUNNER_TIMEOUT_FACTOR% %TEST_RUNNER_EXTRA%
+        run: py -3 test/functional/test_runner.py --jobs $NUMBER_OF_PROCESSORS --ci --quiet --tmpdirprefix="${RUNNER_TEMP}" --combinedlogslen=99999999 --timeout-factor=${TEST_RUNNER_TIMEOUT_FACTOR} ${TEST_RUNNER_EXTRA}
```

Comment:
I see in this job and the `windows-native-test` you have swapped to using powershell rather than cmd. We previously had a problem when doing that that `TEST_RUNNER_EXTRA=''` was causing an error of `WARNING! Test '' not found in full test list.` when running the function tests. See: https://github.com/bitcoin/bitcoin/issues/29534

Why does this work now?

Reply:
> I see in this job and the `windows-native-test` you have swapped to using powershell rather than cmd.

Not `powershell`, but rather `bash`:https://github.com/bitcoin/bitcoin/blob/04ddddec1ed6cd8af060145ba1e85ada16560ef4/.github/workflows/ci.yml#L24-L28

Additionally, you can check the CI [log](https://github.com/bitcoin/bitcoin/actions/runs/13921287455/job/39030781425?pr=31176#step:12:3):
```
   shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
```